contrib/run_function.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+export KUBECONFIG="$(kind get kubeconfig-path)"
+
+kubectl rollout status deploy/gateway -n openfaas
+kubectl port-forward deploy/gateway -n openfaas 8080:8080 &
+
+# Check if Openfaas GW is Up
+for i in {1..60};
```

Comment:
Unlike to be enough?

Reply:
I think this doesn't need to spin wait since `kubectl rollout status` includes the health check.

Reply:
I has removed it in one of my commit, but found out that the test gave some errors about gateway not being up, not really sure why it happened, but seems like port-forward takes some time to warm up. See https://travis-ci.org/openfaas/faas-netes/builds/502995868#L789 from latest build

Reply:
Instead of Checking for up, I've added a sleep step